README.md

```diff
@@ -1,78 +1,99 @@
+You can integrate `TwilioVideo.framework` manually by following these [install instructions](https://www.twilio.com/docs/api/video/download-video-sdks#manual).
 
-This sample app is written in Ojective-C. You will need at least Xcode 8.0 in order to run the application.
+## Quickstart
 
-Review the code to see how how to set up key classes like `TwilioVideo`, `TVIRoom`, `TVIParticipant`, and `TVICameraCapturer`. The ViewController implements the `TVIRoomDelegate`, and `TVIParticipantDelegate` protocols in order to display remote Participant video on screen.
+### Running the Quickstart
 
-## Prerequisites
+To get started with the Quickstart application follow these steps:
 
-This project uses Apple's Objective-C for iOS, and the only supported way to develop iOS apps is on an Apple computer running OS X and Xcode. We have tested this application with the latest versions of iOS (10.0) and Xcode (8.0) at the time of this writing.
+1. Open this `ObjCVideoQuickstart.xcworkspace` in Xcode
 
-You can use the iOS Simulator that comes with Xcode to do your testing, but local video will not be shared since the Simulator cannot access a camera. If you have an iOS device, you can now run apps from Xcode on your device without a paid developer account.
+<img width="700px" src="images/quickstart/xcode-video-quickstart.png"/>
 
-## Twilio Library Setup for the Project
+2. Type in an identity and click on "Generate Access Token" from the [Testing Tools page](https://www.twilio.com/user/account/video/dev-tools/testing-tools).
 
-You will need to add the Twilio Video library to the project to compile and run. You can do this using the [Cocoapods](https://cocoapods.org/) dependency management system. Version 1.0 or greater is required.
+<img width="700px" src="images/quickstart/generate_access_tokens.png"/>
 
-### Using Cocoapods
+Note: If you enter the Room Name, then you can restrict this user's access to the specified Room only. Ideally, you want to implement and deploy an Access Token server to generate tokens. You can read more about setting up your own Access Token Server in this [section](#setup-an-access-token-server). Read this [tutorial](https://www.twilio.com/docs/api/video/user-identity-access-tokens) to learn more about Access Tokens.
 
-First, you will need to have Cocoapods 1.0.0+ installed on your Mac, so go ahead and do that if you haven't already - the directions are here: [Getting Started with Cocoapods](https://guides.cocoapods.org/using/getting-started.html). If you're not sure, type `pod --version` into a command line.
+3. Paste the token you generated in the earlier step in the `ViewController.m`.
 
-Next, just run `pod install` from the command line in the top level directory of this project. Cocoapods will install the Twilio library and then set up a .xcworkspace file that you will use to run your project from now on.
+<img width="700px" src="images/quickstart/xcode-video-quickstart-token.png"/>
```

Comment:
This image looked really blurry to me. Should we use half the retina width (612)?